sdk/lib/rtl/actctx.c

```diff
@@ -9,7 +9,7 @@
  *                  Jacek Caban for CodeWeavers
  *                  Alexandre Julliard
  *                  Stefan Ginsberg (stefan.ginsberg@reactos.org)
- *                  Samuel Serapi�n
+ *                  Samuel Serapión
  */
 
 /* Based on Wine 3.2-37c98396 */
```

Comment:
This reference to the Wine revision should probably be updated.

Reply:
Or it cam be removed at all, since the Wine version is already specified in WINESYNC.TXT and the whole code in this file is now synced to that Wine version, not only a specific part for which this comment was written.